docs/C-Lightning-setup.md

```diff
@@ -101,5 +102,35 @@ If the server started successfully, you should get the below output on the conso
 
 Open your browser at the following address: http://localhost:3000 to access the RTL app.
 
+### <a name="service"></a>Running c-lightning RTL as a service (RPi or Linux Platform)
+In case you are running a headless Rpi or a Linux node, you can configure RTL as a service.
+
+* Create rtl.service systemd unit in and with the following content. Save and exit.
+```
+[Unit]
+Description=RideTheLightning
+Requires=c-lightning-REST.service
```

Comment:
This service dependency is applicable, only if c-lightning-REST is being run separately as a service.

If c-lightning-REST is being run as a plugin, than this dependency is not required.

Reply:
Yes, that is correct. I might look into just setting this up as a c-lightning plugin than have it running as a service.